src/Analyzer.Utilities/CSharpSyntaxGeneratorExtensions.cs

```diff
@@ -1,22 +1,23 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-using System.Composition;
-using System.Diagnostics;
+﻿using System.Diagnostics;
 using System.Linq;
 using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CodeFixes;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
 
-namespace Microsoft.ApiDesignGuidelines.Analyzers
+namespace Analyzer.Utilities
 {
-    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
-    public class CSharpOverrideMethodsOnComparableTypesFixer : OverrideMethodsOnComparableTypesFixer
+    internal static class CSharpSyntaxGeneratorExtensions
     {
-        protected override SyntaxNode GenerateOperatorDeclaration(SyntaxNode returnType, string operatorName, IEnumerable<SyntaxNode> parameters, SyntaxNode notImplementedStatement)
+        internal static SyntaxNode CSharpOperatorDeclaration(
```

Comment:
Cyrus recently [added support](https://github.com/dotnet/roslyn/commit/2933883b0041a987310594b56f2f73f2fadc3263) to `SyntaxGenerator` to generate Operator declarations - so this and the VB code can be deleted and replaced with a call to `SyntaxGenerator.OperatorDeclaration` now.